### PR TITLE
Feature/backend/sanitize message body

### PR DESCRIPTION
**Pull request by sapiens-sapide**

### CODE REVIEW

File: src/backend/interfaces/REST/py.server/caliopen_api/message/message.py [hunk 1]
```diff
@@ -68,17 +68,19 @@ def collection_post(self):
...
+        #     message = ObjectMessage(self.user.user_id, message_id=message_id)
+        #     try:
+        #         message.get_db()
+        #     except Exception as exc:
+        #         log.warn(exc)
+        #         raise ResourceNotFound
+        #
+        #     message.unmarshall_db()
+        #     return message.marshall_json_dict()
+        return Response(None, 301)
```
Comment (gdchamal) [inline src/backend/interfaces/REST/py.server/caliopen_api/message/message.py]: should return to correct v2 route not None

File: src/backend/main/go.backends/index/elasticsearch/index_lookup_test.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ func TestElasticSearchBackend_RecipientsSuggest(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	user_id := "f34528ca-b172-4df3-b27f-32cb18e1ad1f"
+	user_id := "8430b7e1-018e-482e-a06b-eb5e01a9ebcf"
```
Comment (gdchamal) [inline src/backend/main/go.backends/index/elasticsearch/index_lookup_test.go]: it won't be easy to update the user_id any time we reimport. Testing strategy with volatile identifier have to be discussed better.

File: src/backend/interfaces/REST/py.server/caliopen_api/message/message.py (hunk 1 above)
Comment (sapiens-sapide) [inline src/backend/interfaces/REST/py.server/caliopen_api/message/message.py]: Exception raised with commit #0dab108

File: src/backend/main/go.backends/index/elasticsearch/index_lookup_test.go (hunk 2 above)
Comment (sapiens-sapide) [inline src/backend/main/go.backends/index/elasticsearch/index_lookup_test.go]: Yes, definitely. Need to handle user_id & discussion_id with command line args (`go test -args`).

**Pull request merged**